foreman/data_refinery_foreman/gatherer/agents/microarray_ae.py

```diff
@@ -0,0 +1,126 @@
+"""MicroArray ArrayExpress accession gathering automation.
+Data source: https://www.ebi.ac.uk/biostudies/help"""
+
+from typing import List, Set
+
+import requests
+from retrying import retry
+
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_common.models.gathered_accession import GatheredAccession
+from data_refinery_foreman.gatherer.agents.base import AccessionAgentBase
+
+logger = get_and_configure_logger(__name__)
+
+
+class MicroArrayExpressAccessionAgent(AccessionAgentBase):
```

Comment:
Can we update these to just the source name? We should probably add an attribute that specifies the technology that they provide.

ArrayExpressAgent or AEAgent,
GEOAgent,
EBIAgent,

Reply:
I think also we should have the Agent implement an attribute called source_name or something similar and have the command inspect the supported agents for source names

Reply:
Done!